cooked-validators/src/Cooked/MockChain/Wallet.hs

```diff
@@ -5,7 +5,9 @@
 
 module Cooked.MockChain.Wallet where
 
+import qualified Cardano.Api as C
```

Comment:
I think the qualified names in the module are becoming confusing. Maybe we could refactor:
- avoiding 1-2 letters names apart from `Pl`, including `M` which could be imported as `Map` along with a non qualified import for the type `Map`
- making it clear what comes from Plutus and what comes from Cardano.
There are two `Crypto` modules here, one from Plutus, one from Cardano.
- being consistent whether we name the submodule (e.g. `Validation`/`Crypto`/`CW`), give them the lib name (`Pl`/`Cardano`), or both (e.g. `PlCrypto`/`CardanoCrypto`).